packages/hint-typescript-config/src/helpers/config-checker.ts

```diff
@@ -15,6 +15,38 @@ const findValue = (property: string, config: TypeScriptConfig) => {
     return current;
 };
 
+const findLocation = (propertyPath: string, targetValue: boolean | string, originalConfig: TypeScriptConfig, getLocation: IJSONLocationFunction) => {
+    const valueInOriginal = findValue(propertyPath, originalConfig);
+
+    let location = null;
+
+    const ancestors = propertyPath.split('.');
+
+    ancestors.pop();
+
+    if (typeof valueInOriginal !== 'undefined') {
+
+        location = getLocation(propertyPath, { at: 'value' });
+
+    } else if (targetValue !== valueInOriginal &&
```

Comment:
Given `valueInOriginal` is `undefined` by this point do we even need to compare?
```suggestion
    } else if (
```

Reply:
Done